src/lib/stores/billing.ts

```diff
@@ -183,18 +183,23 @@ export function checkForTrialEnding(org: Organization) {
     }
 }
 
-export function checkForUsageLimit(org: Organization) {
+export async function checkForUsageLimit(org: Organization) {
     if (!org?.billingLimits) {
         readOnly.set(false);
         return;
     }
     const { bandwidth, documents, executions, storage, users } = org.billingLimits;
```

Comment:
```suggestion
    const { bandwidth, documents, executions, storage, users } = org?.billingLimits;
```
Don't we need conditional chaining in all the places ?